src/orders/views/OrderTransactionRefundCreate/OrderTransactionRefundCreate.tsx

```diff
@@ -41,8 +43,15 @@ const OrderTransactionRefund: React.FC<OrderTransactionRefundCreateProps> = ({ o
             submitData.orderGrantRefundCreate.grantedRefund!.id,
           ),
         );
+      } else {
+        handleCreateRefundErrors({
+          submitData,
+          notify,
+          setLinesErrors,
+        });
```

Comment:
if you touch this block, please do some short refactoring, adding an "else" statement doesn't make it more readable, so create a function that handles the entire "onComplete" logic and use "exit earlier" pattern (void return call) to avoid unnecessary nesting